### ISSUE TITLE
Detect regressions in stability

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt.fyi#899]
With https://github.com/web-platform-tests/wpt/issues/14165 resolved, we'll have more PRs blocked on failing stability jobs in Taskcluster. Understanding what is even going on can be hard, as the reason is buried in the Taskcluster logs.

Some ideas that one could put together to address this:
- extract information from the *-stability jobs in the custom status checks
- look at recent history of the tests reported as flaky
- a button that says "no this is not my fault"

Also, maybe, a more systematic way of detecting flakiness over time? Perhaps once a day we could run all tests twice, so that we know for sure that differences are due to flakiness and not test changes?

**Comment by gsnedders** [comment]
related to #66, no?

**Comment by foolip** [comment]
Yes, but at least when I filed https://github.com/web-platform-tests/wpt.fyi/issues/66 I was thinking of flakiness in a series of master runs, where for this I was thinking comparing flakiness detected by Taskcluster on a PR to the flakiness determined from master runs.
